src/flamenco/snapshot/fd_snapshot_http.c

```diff
@@ -75,19 +76,13 @@ fd_snapshot_http_new( void *               mem,
   /* Left-aligned render the headers, completing the message  */
 
   char * p = fd_cstr_init( this->req_hdrs );
-  static char const hdr_part1[] =
+  static char const hdr[] =
     " HTTP/1.1\r\n"
     "user-agent: Firedancer\r\n"
     "accept: */*\r\n"
-    "host: ";
-  p = fd_cstr_append_text( p, hdr_part1, sizeof(hdr_part1)-1 );
-
-  p = fd_cstr_append_printf( p, FD_IP4_ADDR_FMT ":%u",
-                             FD_IP4_ADDR_FMT_ARGS( dst_ipv4 ), dst_port );
-  static char const hdr_part2[] =
-    "\r\n"
+    "host: %s\r\n"
     "\r\n";
-  p = fd_cstr_append_text( p, hdr_part2, sizeof(hdr_part2)-1 );
+  p = fd_cstr_append_printf( p, hdr, dst_str );
```

Comment:
Please avoid use of unsafe printf. This isn't bounds checked

Reply:
(Was fine for IPv4 because the longest IP address is still pretty short, but more scary for a user provided string via redirects)